seat.c

```diff
@@ -835,7 +854,10 @@ seat_set_focus(struct cg_seat *seat, struct cg_view *view)
 
 	view_activate(view, true);
 	char *title = view_get_title(view);
-	output_set_window_title(server->output, title);
+	struct cg_output *output;
+	wl_list_for_each(output, &server->outputs, link) {
+		output_set_window_title(output, title);
```

Comment:
Maybe we should move the logic of iterating over all outputs in order to set their title(s) to `output_set_window_title`, what do you think?

Reply:
I could add a `desktop_set_window_title` convenience wrapper. The name of `output_set_window_title` implies that it's a method for an output instance, though.

Reply:
On second thought, can it be that a window is only on one output? Think e.g. a dialog window, which is currently centered on Cage's single output.

In that case, this for-loop as it is will set the window title on all outputs. Arguably, in case of a dialog window, that is still correct. I'm just thinking if a check whether the focused view is on the output whose title we're setting should be added? E.g. `if (view_is_on_output) { output_set_window_title(...) }`.

Reply:
Done, although the `view_is_on_output` part is a wee bit clumsier than your psuedo-code as it involves intersecting the surface box for a view with all output boxes within the current layout and seeing if the intersect is zero-sized.

Also, it is somewhat hit-and-miss: It only gets it "right" if a view doesn't change its size (and thus output coverage) after focus calculation. In my tests, alacritty is consistently only big enough to cover one of my outputs when focus is set, but scales up to all outputs immediately afterwards. Thus, the output title is only set on the first output until focus changes away and back.

Reply:
Hm, in that case, let's see how we fare without this bit of code :) Sorry!

Reply:
No worries, it was pretty easy to try. :)

Reply:
Thanks!